sample/build.gradle

```diff
@@ -15,6 +15,12 @@ android {
     sourceCompatibility JavaVersion.VERSION_1_7
     targetCompatibility JavaVersion.VERSION_1_7
   }
+  buildTypes {
+    release {
+      minifyEnabled true
```

Comment:
I think that you need some proguard rules here, at least for the Android classes.